index.bs

```diff
@@ -673,7 +682,10 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
   <dd>This reflects the current <a>contrast</a> setting of the camera.</dd>
 
   <dt><dfn dict-member for="MediaTrackSettings"><code>pan</code></dfn></dt>
-  <dd>This reflects the current <a>pan</a> setting of the camera.</dd>
+  <dd>This reflects the current <a>pan</a> setting of the camera.
+
+  If the track has been created without <a>requesting permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true or if that permission request is denied, the track does not support <a>pan</a>.
```

Comment:
It seems checking the permission value at the time getSettings is called is sufficient.
Why adding a check for the permission at the time the track was created?

Reply:
This PR does not add a check for the permission at the time the track was created. That check already exists in the spec. So reusing the result of that check (or the fact that the check was skipped because PTZ was not requested) seems sufficient.

Checking the permission value at the time getCapabilities or getSettings is called breaks the developer assumptions that availability of capabilities and settings do not change during track life time (like it is common to create UI components and their event listeners based on capabilities when the track starts).

I also think that PTZ capabilities and settings should not suddenly appear to non-PTZ tracks just because the origin gains PTZ permission through another getUserMedia call during the life time of the non-PTZ track.

What happens if the PTZ permission is revoked is more interesting. One option is that the track could be ended (in which case getCapabilities and getSettings do not have to check permissions). The second option is that the settings would not be updated after the revocation (in which case getCapabilities and getSettings again do not have to check permissions). The third option is that PTZ capabilities and settings would be removed (but would that require some kind of event so that the web page could listen for that event?).

Reply:
Let's continue this discussion at https://github.com/w3c/mediacapture-image/pull/249